test/functional/mempool_limit.py

```diff
@@ -366,7 +370,9 @@ def run_test(self):
         assert_greater_than(worst_feerate_btcvb, (parent_fee + child_fee) / (tx_parent_just_below["tx"].get_vsize() + tx_child_just_above["tx"].get_vsize()))
         assert_greater_than(mempoolmin_feerate, (parent_fee) / (tx_parent_just_below["tx"].get_vsize()))
         assert_greater_than((parent_fee + child_fee) / (tx_parent_just_below["tx"].get_vsize() + tx_child_just_above["tx"].get_vsize()), mempoolmin_feerate / 1000)
-        assert_raises_rpc_error(-26, "mempool full", node.submitpackage, [tx_parent_just_below["hex"], tx_child_just_above["hex"]])
+        res = node.submitpackage([tx_parent_just_below["hex"], tx_child_just_above["hex"]])
+        for wtxid in [tx_parent_just_below["wtxid"], tx_child_just_above["wtxid"]]:
+            res["tx-results"][wtxid]["error"] == "mempool full"
```

Comment:
Are we trying to assert that this is true? Pretty sure this just keeps going if the expression is False
```suggestion
            assert_equal(res["tx-results"][wtxid]["error"], "mempool full")
```

Reply:
oops, taken